src/hooks/tradingView/useChartMarketAndResolution.ts

```diff
@@ -21,32 +21,34 @@ export const useChartMarketAndResolution = ({
   currentMarketId,
   isViewingUnlaunchedMarket,
   tvWidget,
-  isWidgetReady,
   savedResolution,
 }: {
   currentMarketId: string;
   isViewingUnlaunchedMarket?: boolean;
   tvWidget: TvWidget | null;
-  isWidgetReady?: boolean;
   savedResolution?: ResolutionString;
 }) => {
   const dispatch = useAppDispatch();
 
   const selectedResolution: string =
     useAppSelector((s) => getSelectedResolutionForMarket(s, currentMarketId)) ?? DEFAULT_RESOLUTION;
 
-  const chart = isWidgetReady ? tvWidget?.chart() : undefined;
+  const chart = tvWidget?._ready ? tvWidget.chart() : undefined;
```

Comment:
its kind of strange that we're also using tvWidget.chart().resolution as a react prop. we should probably add a subscribe to `onIntervalChanged` to run callback stuff that way instead

Reply:
Add in this PR or future one?

Reply:
Future one